lms/static/sass/shared/_footer.scss

```diff
@@ -190,6 +190,10 @@
       }
     }
   }
+
+  @media print {
```

Comment:
That would be neat, but would require changing the page template rather than a minimal CSS change. The header still includes the platform logo, platform name, and site name (see image below).